### DISCUSSION TITLE
Bluetooth headset support

### DISCUSSION THREAD

**Discussion by gdyuldin** [gdyuldin/x6100_gui#15] [Ideas]
Some additional features might be implemented after:
* VOX
* Compressor

**Comment by dk5cr** [comment]
Hello Georgy, 
i would appreciate, if you could integrate a VOX-function into x6100 firmware/gui. Especially for communication with some cellphone-apps a VOX is strictly necessary for USB-D in FT8, FT4 and others. Only few apps (like FT8CN) provide an own CAT-Interface with different config-files for the whole bunch of rigs existing on the market. 
Thank youuuuu!!!! 
BTW: Happy New Year :-) 
73 de Cornel, DK5CR

**Comment by KbCr2iKaJnQ** [comment]
I'm also interested in VOX control for x6100,... in possibly an extended manual or other document, I noticed control codes,...

pgs118-119
(vertical "|" pipe lines represent table boxes)
0x14 | 0x16 | --- | Get VOX Gain
0x14 | 0x17 | --- | Get Anti-VOX Gain
0x14 | 0x16 | BCD code | Set VOX Gain
0x14 | 0x17 | BCD code | Set Anti-VOX Gain

0x16 | 0x46 | --- | Get VOX Switch
0x16 | 0x46 | 0x00 | VOX off
0x16 | 0x46 | 0x01 | VOX on


I don't know how to implement these codes,...
And don't know if they can be selected get control from the micro(s), or digi input line

It would be nice to have a "VOX-(M,D)" indicator on the screen,...
And, F1 or F2 button from mic, or other control key on main unit, to control VOX on/off, (vox-m, vox-d if possible also)

gain and anti gain could be menu items

But I don't know if the hardware supports it.

I think it might also be possible, to have an subroutine be able to "Listen/sample"  input source, and trigger PTT command with time delay to release ptt after source has reduced below adjustable value of gain.

All programming I don't know how to implement,... 

I don't think the VOX commands work, because I think I've seen these controls in WFVIEW and FLRIG, or similar rig control, and they  don't change any function on the x6100,...

**Reply by gdyuldin** [reply]
VOX is not implemented on the BASE firmware.
